docs/platform/howto/attach-vpc-aws-tgw.md

```diff
@@ -1,130 +1,170 @@
+peering connection.
+
+:::note
+In the request, you can use the `--user-peer-network-cidr` argument multiple times to
+define more than one peer network CIDR. You can also create the attachment without a CIDR,
+but you would need to add it later to make your attachment operational.
+:::
+
+Depending on the type of your Aiven VPC, select **Project VPC** or **Organization VPC**,
+and run the provided commands, replacing `$` placeholders with meaningful values.
+
+   <Tabs groupId="group1">
+   <TabItem value="pj-vpc" label="Project VPC" default>
+```bash
 avn vpc peering-connection create       \
-  --project-vpc-id $aws_vpc             \
+  --project-vpc-id $vpc_id              \
   --peer-cloud-account $user_account_id \
   --peer-vpc $user_tgw_id               \
   --user-peer-network-cidr $user_peer_network_cidr
 ```
+   </TabItem>
+   <TabItem value="org-vpc" label="Organization VPC">
+```bash
+avn organization vpc peering-connection create \
+  --organization-vpc-id $vpc_id                \
+  --peer-cloud-account $user_account_id        \
+  --peer-vpc $user_tgw_id                      \
+  --user-peer-network-cidr $user_peer_network_cidr
+```
+   </TabItem>
+   </Tabs>
 
-The state of the Aiven peering connection is `APPROVED`.
-
-:::note
-You can use the `--user-peer-network-cidr` argument multiple
-times to define more than one peer network CIDR. You can also
-create the attachment without any CIDRs and add them later. In this case, the
-attachment will be not be of any use until that is done since no
-addresses will be routed through the TGW from the Project VPC.
-:::
-
-### Accept AWS Transit Gateway VPC attachment
+Once submitted, the request gets the `APPROVED` status, and the Aiven Platform starts
+building the connection by creating the AWS Transit Gateway VPC attachment, which can take
+a few minutes. Once the connection is built, the status changes to `PENDING_PEER`.
 
-Once the Aiven platform has built the connection by creating an AWS Transit Gateway VPC attachment,
-the state changes to `PENDING_PEER` if everything went well. Otherwise
-the state information will indicate why the attachment failed to be
-created. It can take a few minutes to build the attachment.
+### Check the connection status
 
-To check the state, run:
+To check the status of your peering connection request, select either **Project VPC** or
+**Organization VPC**, depending on your Aiven VPC type, and run the provided commands,
+replacing `$` placeholders with meaningful values.
 
-```shell
+   <Tabs groupId="group1">
+   <TabItem value="pj-vpc" label="Project VPC" default>
+```bash
 avn vpc peering-connection get          \
-  --project-vpc-id $project_vpc_id      \
+  --project-vpc-id $vpc_id              \
   --peer-cloud-account $user_account_id \
   --peer-vpc $user_tgw_id -v
 ```
+   </TabItem>
+   <TabItem value="org-vpc" label="Organization VPC">
+```bash
+avn organization vpc peering-connection get \
```

Comment:
A command like this does not exists in the CLI PR here https://github.com/aiven/aiven-client/pull/399/files . `organization vpc get` also shows properties and states of peering connections, so that can be used.

And like other organization VPC commands, it requires `--organization-id`.